pom.xml

```diff
@@ -161,9 +163,7 @@
             <plugin>
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-javadoc-plugin</artifactId>
-              <configuration>
-                <additionalparam>${beam.javadoc_opts}</additionalparam>
-              </configuration>
+	      <version>2.10.3</version>
```

Comment:
claim: it is profile managed, but the ordering of the pom (`profiles` before `build/pluginManagement`) makes this non-obvious.